docs/source/indexes/cagra.rst

```diff
@@ -98,53 +98,148 @@ Tuning Considerations
 The 3 hyper-parameters that are most often tuned are `graph_degree`, `intermediate_graph_degree`, and `itopk_size`.
 
 Memory footprint
-----------------
+================
 
-CAGRA builds a graph that ultimately ends up on the host while it needs to keep the original dataset around (can be on host or device).
+CAGRA stores the dataset (raw vectors) and nearest-neighbor graph (neighbor IDs) in memory.
+The dataset can be on **host or device**; the graph is **pinned to host memory**.
```

Comment:
Help me understand this a bit- why is this saying the graph is in pinned host memory? I don't see indication in the code that this is true.